index.html

```diff
@@ -38,8 +38,11 @@
     </header>
     <main>
         <section class="section section-main">
+            <img class="image-section"
+                src="https://cdn.shortpixel.ai/spai/q_+ret_img+to_webp/https://www.banarasisaree.com/wp-content/uploads/2021/11/slider-home-wedding-2nw.jpg"
+                alt="">
```

Comment:
whenever you have a time then use a <figure> element to mark up a photo in a document . eg . <figure> <img/> <figure> . alt should have value.